allofplos/corpus/corpus_class.py

```diff
@@ -9,10 +9,8 @@
 class Corpus():
     """A collection of PLOS articles."""
 
-    def __init__(self, directory=None, extension='.xml'):
+    def __init__(self, directory=get_corpus_dir(), extension='.xml'):
```

Comment:
This is not a pattern that I support having, __anywhere__ in the codebase. The set the default to `None` and then modify it if it is `None` is a classic pythonic  for addressing this kind of case. 

If you want a description of why we shouldn't do this, check out: 
the "Default arguments evaluation" section of  http://blog.thedigitalcatonline.com/blog/2015/02/11/default-arguments-in-python/.

Reply:
> default arguments are evaluated at definition time (for example when you first import the module)

meaning that it's no longer going to be appropriately flexible for people who don't want to use a environmental variable directly.

Reply:
I see it now, so I will revert this change.